core/src/main/java/com/linecorp/armeria/client/retry/RetryStrategy.java

```diff
@@ -34,7 +35,11 @@
 /**
  * Determines whether a failed request should be retried.
  * If you need to determine by looking into the {@link Response}, use {@link RetryStrategyWithContent}.
+ *
+ * @deprecated Use {@link RetryRule} directly or convert the existing {@link RetryStrategy} to {@link RetryRule}
```

Comment:
If `RetryStrategy` is deprecated, I don't think we need to mention `toRetryRule()` in the deprecation message

Reply:
Yeap, let me move `toRetryRule()` to internal.